core/api/src/config/sequelize.ts

```diff
@@ -62,6 +62,14 @@ export const DEFAULT = {
       password: password,
       models: [join(__dirname, "..", "models")],
       migrations: [join(__dirname, "..", "migrations")],
+      pool: {
+        max: process.env.SEQUELIZE_POOL_SIZE
+          ? parseInt(process.env.SEQUELIZE_POOL_SIZE)
+          : 5,
+        min: 0,
+        acquire: 30000,
+        idle: 10000,
+      },
```

Comment:
I don't think the average Grouparoo cluster will need to change the timeouts... but the pool size seems likely.